lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -232,6 +237,15 @@ class AwsCompileKafkaEvents {
           kafkaResource.Properties.Enabled = enabled;
         }
 
+        if (consumerGroupId) {
+          kafkaResource.Properties.SelfManagedKafkaEventSourceConfig = {
+            ConsumerGroupId: consumerGroupId,
+          };
+          kafkaResource.Properties.AmazonManagedKafkaEventSourceConfig = {
```

Comment:
This shouldn't be here - there's a separate event for MSK under https://github.com/serverless/serverless/blob/main/lib/plugins/aws/package/compile/events/msk/index.js

Reply:
Ah I hadn't realised there was a separate MSK event, will move it here 👍